builtin/credential/ldap/backend.go

```diff
@@ -183,13 +218,17 @@ func getBindDN(cfg *ConfigEntry, c *ldap.Conn, username string) (string, error)
 		if cfg.UPNDomain != "" {
 			bindDN = fmt.Sprintf("%s@%s", EscapeLDAPValue(username), cfg.UPNDomain)
 		} else {
+			// When is this ever a valid bind string?
```

Comment:
Please remove this comment. If you want to argue against why this line should be there, that's fine, but let's keep that in the discussion.